testpackage/tests/Flowthrough_damr/Flowthrough_damr.cfg

```diff
@@ -0,0 +1,116 @@
+ParticlePopulations = proton
+
+project = Flowthrough
+propagate_field = 1
+propagate_vlasov_acceleration = 1
+propagate_vlasov_translation = 1
+dynamic_timestep = 0
```

Comment:
Is that intended?

Reply:
Turning off dyndt? yes.

Reply:
OK. Is it needed? We're possibly catching more potential bugs if it's dynamic, no? But fine by me if it's static.